cinema/models.py

```diff
@@ -1,10 +1,62 @@
 from django.db import models
+from rest_framework.exceptions import ValidationError
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=63)
+    last_name = models.CharField(max_length=63)
+
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
 
 
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="movies")
 
     def __str__(self):
         return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> str:
+        return self.name
+
+    def clean(self):
+        if self.rows <= 0:
+            raise ValidationError({
+                "row": "Rows must be greater then 0"
```

Comment:
```suggestion
                "row": "Rows must be greater than 0"
```
just a small suggestion